### PR TITLE
TF-435 Scaled cursor position with device pixel ratio

### PR DESCRIPTION
**Pull request by GabrielHomewoodUL**
- [x] Code Review 
- [x] Developer Testing: Key item tests 
- [ ] Product Owner Review (Optional) 
- [ ] Update Release Notes on the JIRA ticket for harvest at release time

### CODE REVIEW

File: TF_Tooling_Web/src/TouchFreeToolingTypes.ts [hunk 1]
```diff
@@ -51,15 +51,16 @@ export class TouchFreeInputAction {
 // Used to translate the raw actions that come across the websocket (<WebsocketInputActions>) and
 // convert them into the Tooling-friendly <TouchFreeInputAction> format.
 export function ConvertInputAction(_wsInput: WebsocketInputAction): TouchFreeInputAction {
-    const yPosition = window.innerHeight - _wsInput.CursorPosition.y;
+    const yPosition = (window.innerHeight - _wsInput.CursorPosition.y) / window.devicePixelRatio;
```
Comment (PeterNancollis-UH) [inline TF_Tooling_Web/src/TouchFreeToolingTypes.ts]: This is doing the operations in the wrong order, so the cursor ends up offset as we subtract the unscaled window height

Comment (GabrielHomewoodUL) [reply TF_Tooling_Web/src/TouchFreeToolingTypes.ts]: I have updated this!

Comment (PeterNancollis-UH) [reply TF_Tooling_Web/src/TouchFreeToolingTypes.ts]: Working great after the fix, the y position of the cursor is consistent at different scalings

**Comment by PeterNancollis-UH** [comment]
A knock of effect of changing the position scaling is that the cursor animation is now misaligned when the screen is scaled, so it shrinks towards the bottom corner rather than the middle

chrome_7A9SXD70cW

**Review by NatGreenwood-UL — APPROVED** [review]
I've confirmed that the offset experienced by Pete above is no longer present and am happy with the code 👍

**Pull request merged**